sdk/authorization/mgmt/src/main/java/com/azure/management/graphrbac/AddOwnerParameters.java

```diff
@@ -15,7 +13,11 @@
 public final class AddOwnerParameters {
     /*
      * A owner object URL, such as
-     * "https://graph.windows.net/0b1f9851-1bf0-433f-aec3-cb9272f093dc/directoryObjects/f260bbc4-c254-447b-94cf-293b5ec434dd", where "0b1f9851-1bf0-433f-aec3-cb9272f093dc" is the tenantId and "f260bbc4-c254-447b-94cf-293b5ec434dd" is the objectId of the owner (user, application, servicePrincipal, group) to be added.
+     * "https://graph.windows.net/0b1f9851-1bf0-433f-aec3-cb9272f093dc/
+     * directoryObjects/f260bbc4-c254-447b-94cf-293b5ec434dd",
```

Comment:
Single word URL without space is ignored in line length check and should not be broken.

Reply:
Not sure. But I got error for the line length so I changed it.

Reply:
Make sure you delete sdk-build-tool from local maven and re-install it.

https://teams.microsoft.com/l/message/19:885436a8c3a64adb87a6c5817b07187b@thread.skype/1586308673100?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=de995b1a-16c1-4cc7-a49b-bc9f90bc6acd&parentMessageId=1586226373590&teamName=Azure%20Management%20Experience&channelName=Fluent&createdTime=1586308673100